programmer-skills/start-to-develop-js.md

```diff
@@ -26,10 +26,10 @@ In this README I'm going to talk about two main kinds of code editors:
 
 Sometimes, it's tricky to setup your environment to start developing small projects, or concept proofs. Maybe you only want to be able to
 share a small snippet with other people. To accomplish those things we
-have available some online code editors like [jsbin](https://jsbin.com),
+have some online code editors available like [jsbin](https://jsbin.com),
 [jsfiddle](https://jsfiddle.net/), or [codepen](http://codepen.io/).
 There you can write your code and run it without leaving the browser.
-There are too others tools with more features like
+Other tools with more features can also be found like
```

Comment:
Nice catch. For clarity's sake and avoiding having long run-off sentence I would propose this commit:

> There are also other tools with more features. For instance, [cloud9](https://c9.io/) allows you to run code from other languages - and even run a web server in nodejs.